src/libraries/System.Transactions.Local/tests/TransactionScopeTest.cs

```diff
@@ -507,6 +507,7 @@ public void RMFail1()
 
         [Fact]
         [OuterLoop] // 30 second timeout
+        [PlatformSpecific(~TestPlatforms.Browser)] // Cannot wait on monitors on this runtime.
```

Comment:
-> PlatformDetection.IsThreadingSupported

Reply:
Addressed in recent commits